modules/core/src/main/scala/org/scalasteward/core/git/package.scala

```diff
@@ -30,7 +30,10 @@ package object git {
     val base = baseBranch.fold("")(branch => s"${branch.name}/")
     update.on(
       update = u => Branch(s"$updateBranchPrefix/$base${u.name}-${u.nextVersion}"),
-      grouped = g => Branch(s"$updateBranchPrefix/$base${g.name}")
+      grouped = g => {
+        val uniqueness = Math.abs(g.updates.map(_.nextVersion).sortBy(_.value).hashCode())
```

Comment:
Not a bad idea, however I believe this should be a default because the behaviour is now similar to Update.Single. 
WDYT?

Reply:
🤔 I don't think that should be the default value since it will mean we are breaking compatibility (people already using it won't have the same branch anymore).

Regarding how to add the configuration, adding more options will make the config more complicated, while adding a magic string is easy and something people are already familiar with on other parts of the configuration.

Also take into account this is something people can already do on their own, for example, if using GitHub Actions, you can add a super-simple step to update the configuration for you on every run (essentially creating a new value each time):

```yaml
on:
  schedule:
    - cron: '0 0 * * 0'

name: Launch Scala Steward

jobs:
  scala-steward:
    runs-on: ubuntu-22.04
    name: Launch Scala Steward
    steps:
      - name: Checkout project
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0

      - name: Update `.scala-steward.conf`
         run: timestamp=$(date +%s); sed -i '' -e "s/{{timestamp}}/$timestamp/g" .github/.scala-steward.conf

      - name: Launch Scala Steward
         uses: scala-steward-org/scala-steward-action@v2
         with:
           github-app-id: ${{ secrets.APP_ID }}
           github-app-installation-id: ${{ secrets.APP_INSTALLATION_ID }}
           github-app-key: ${{ secrets.APP_PRIVATE_KEY }}
```

Reply:
I see, alright, I'll add the `hash` variable to the grouping names.